easybuild/easyconfigs/p/python-Levenshtein/python-Levenshtein-0.12.0-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'python-Levenshtein'
+version = '0.12.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.org/project/python-Levenshtein/'
+description = 'Python extension for computing string edit distances and similarities.'
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['033a11de5e3d19ea25c9302d11224e1a1898fe5abd23c61c7c360c25195e3eb1']
+
+dependencies = [
+    ('Python', '3.6.6'),
+]
+
```

Comment:
@timeu Any reason not to `use pip` with this one?

Reply:
No reason good. 
I think I just forgot it to add it. I added them